blocks/editable/index.js

```diff
@@ -58,6 +60,16 @@ function isLinkBoundary( fragment ) {
 		fragment.childNodes[ 0 ].text[ 0 ] === '\uFEFF';
 }
 
+function getLinkProperties( element ) {
+	return { value: element.getAttribute( 'href' ) || '', target: element.getAttribute( 'target' ) || '', node: element };
+}
+
+function getFormatProperties( formatName, parents ) {
+	return formatName === 'link' ? getLinkProperties( find( parents, node => node.nodeName.toLowerCase() === 'a' ) ) : {};
+}
```

Comment:
Minor and personal preference: I would have used a single function and a switch on `formatName`. It makes it easy to extend if needed but it's not very important.

Reply:
👍